<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="7" failed="0" ignored="0" total="8" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="My suite" duration-ms="0" started-at="2018-12-25T15:06:43Z" finished-at="2018-12-25T15:06:43Z">
    <groups>
    </groups>
  </suite> <!-- My suite -->
  <suite name="Portfolio" duration-ms="908" started-at="2018-12-25T15:05:50Z" finished-at="2018-12-25T15:05:51Z">
    <groups>
    </groups>
    <test name="LoginTest" duration-ms="776" started-at="2018-12-25T15:05:50Z" finished-at="2018-12-25T15:05:51Z">
      <class name="com.selenium.framework.datadriven.Portfolio.LoginTest">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="initLogs" is-config="true" duration-ms="34" started-at="2018-12-25T15:05:50Z" finished-at="2018-12-25T15:05:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initLogs -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="loginTest" duration-ms="77" started-at="2018-12-25T15:05:51Z" data-provider="PortFolioTestDataProvider" finished-at="2018-12-25T15:05:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedResult=Success, Password=C13, Runmode=N, UserName=C12, Browser=Mozilla, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test LoginTest inside the suite PortFolioTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test LoginTest inside the suite PortFolioTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.Portfolio.LoginTest.loginTest(LoginTest.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="close" is-config="true" duration-ms="2" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="loginTest" duration-ms="54" started-at="2018-12-25T15:05:51Z" data-provider="PortFolioTestDataProvider" finished-at="2018-12-25T15:05:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedResult=Success, Password=Pass#123, Runmode=Y, UserName=rabindrabiswal404@gmail.com, Browser=Chrome, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test LoginTest inside the suite PortFolioTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test LoginTest inside the suite PortFolioTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.Portfolio.LoginTest.loginTest(LoginTest.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="close" is-config="true" duration-ms="0" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="loginTest" duration-ms="58" started-at="2018-12-25T15:05:51Z" data-provider="PortFolioTestDataProvider" finished-at="2018-12-25T15:05:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedResult=Success, Password=C33, Runmode=N, UserName=C32, Browser=IE, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test LoginTest inside the suite PortFolioTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test LoginTest inside the suite PortFolioTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.Portfolio.LoginTest.loginTest(LoginTest.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="close" is-config="true" duration-ms="0" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="SKIP" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="loginTest" duration-ms="59" started-at="2018-12-25T15:05:51Z" data-provider="PortFolioTestDataProvider" finished-at="2018-12-25T15:05:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExpectedResult=Failure, Password=C43, Runmode=N, UserName=C42, Browser=Edge, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test LoginTest inside the suite PortFolioTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test LoginTest inside the suite PortFolioTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.Portfolio.LoginTest.loginTest(LoginTest.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LoginTest@4dfa3a9d]" name="close" is-config="true" duration-ms="1" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.selenium.framework.datadriven.Portfolio.LoginTest -->
    </test> <!-- LoginTest -->
    <test name="Least Pe rAssetTest" duration-ms="96" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
      <class name="com.selenium.framework.datadriven.Portfolio.LeastPerAssetTest">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LeastPerAssetTest@56ef9176]" name="initLogs" is-config="true" duration-ms="1" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initLogs -->
        <test-method status="SKIP" signature="leastPerAssetTest(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LeastPerAssetTest@56ef9176]" name="leastPerAssetTest" duration-ms="51" started-at="2018-12-25T15:05:51Z" data-provider="PortFolioTestDataProvider" finished-at="2018-12-25T15:05:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Browser=Chrome, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test LeastPerAssetTest inside the suite PortFolioTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test LeastPerAssetTest inside the suite PortFolioTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.Portfolio.LeastPerAssetTest.leastPerAssetTest(LeastPerAssetTest.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- leastPerAssetTest -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.Portfolio.LeastPerAssetTest@56ef9176]" name="close" is-config="true" duration-ms="0" started-at="2018-12-25T15:05:51Z" finished-at="2018-12-25T15:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.selenium.framework.datadriven.Portfolio.LeastPerAssetTest -->
    </test> <!-- Least Pe rAssetTest -->
  </suite> <!-- Portfolio -->
  <suite name="Stock" duration-ms="51074" started-at="2018-12-25T15:05:52Z" finished-at="2018-12-25T15:06:43Z">
    <groups>
    </groups>
    <test name="AddStock" duration-ms="51074" started-at="2018-12-25T15:05:52Z" finished-at="2018-12-25T15:06:43Z">
      <class name="com.selenium.framework.datadriven.StockSuite.AddStock">
        <test-method status="PASS" signature="initLogs()[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="initLogs" is-config="true" duration-ms="3" started-at="2018-12-25T15:05:52Z" finished-at="2018-12-25T15:05:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initLogs -->
        <test-method status="PASS" signature="AddStock(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="AddStock" duration-ms="50203" started-at="2018-12-25T15:05:52Z" data-provider="StockSuiteTestDataProvider" finished-at="2018-12-25T15:06:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Price=100, Runmode=Y, Stock Name=HDFC Bank Ltd., Quantity=10, PurchaseDate=10/12/2014, Browser=Chrome, Result=}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddStock -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="close" is-config="true" duration-ms="727" started-at="2018-12-25T15:06:42Z" finished-at="2018-12-25T15:06:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="SKIP" signature="AddStock(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="AddStock" duration-ms="43" started-at="2018-12-25T15:06:43Z" data-provider="StockSuiteTestDataProvider" finished-at="2018-12-25T15:06:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Price=150, Runmode=N, Stock Name=Tata Consultancy Services Ltd., Quantity=11, PurchaseDate=10/12/2015, Browser=Chrome, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test AddStock inside the suite StockSuiteTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test AddStock inside the suite StockSuiteTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.StockSuite.AddStock.AddStock(AddStock.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddStock -->
        <test-method status="SKIP" signature="AddStock(java.util.Hashtable)[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="AddStock" duration-ms="55" started-at="2018-12-25T15:06:43Z" data-provider="StockSuiteTestDataProvider" finished-at="2018-12-25T15:06:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Price=300, Runmode=N, Stock Name=NTPC Ltd., Quantity=12, PurchaseDate=10/12/2016, Browser=Chrome, Result=}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test AddStock inside the suite StockSuiteTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test AddStock inside the suite StockSuiteTest
at com.selenium.framework.datadriven.TestBase.validateRunmodes(TestBase.java:84)
at com.selenium.framework.datadriven.StockSuite.AddStock.AddStock(AddStock.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddStock -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="close" is-config="true" duration-ms="0" started-at="2018-12-25T15:06:43Z" finished-at="2018-12-25T15:06:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.selenium.framework.datadriven.StockSuite.AddStock@3159c4b8]" name="close" is-config="true" duration-ms="0" started-at="2018-12-25T15:06:43Z" finished-at="2018-12-25T15:06:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.selenium.framework.datadriven.StockSuite.AddStock -->
    </test> <!-- AddStock -->
  </suite> <!-- Stock -->
</testng-results>
